src/Plugin/Field/FieldFormatter/VideoFormatter.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+namespace Drupal\itk_video\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ImmutableConfig;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\itk_video\Plugin\Field\FieldType\Video;
+use Drupal\link\Plugin\Field\FieldFormatter\LinkFormatter;
+use Drupal\itk_video\SupportedVideoProviders;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of the 'itk_video_formatter' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "itk_video_formatter",
+ *   module = "itk_video",
+ *   label = @Translation("Video formatter"),
+ *   field_types = {
+ *     "itk_video_field"
+ *   }
+ * )
+ */
+final class VideoFormatter extends LinkFormatter {
+
+  public function __construct(
+    $plugin_id,
+    $plugin_definition,
+    FieldDefinitionInterface $field_definition,
+    array $settings,
+    $label,
+    $view_mode,
+    array $third_party_settings,
+    protected $urlResolver,
+    protected $httpClient,
+    // @phpstan-ignore property.phpDocType
+    protected $pathValidator,
```

Comment:
It seems that `$pathValidator` is not used in this class. If that's true, it should behandled like the other arguments that are passed through to the parent:


```suggestion
    $pathValidator,
```

(and move it up to the other parent arguments).